ldaptor/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "16.0.0"
+__version__ = "16.0.1"
```

Comment:
please don't bump the version number, that's out job. ;)

if we need to backport to a 16.0, then we'll handle that too. :)

Reply:
thank you for your feedback!